README.md

```diff
@@ -46,8 +46,10 @@ Branch | Build status | Dev site | HTTP Basic auth
   * `cd docroot`
   * `drush si minimal --existing-config`
   * Login to new site:
+  * read admin password from console, or
   * `drush uli`
 * If you want to have some default content for your development, 
+  * `drush csim default_content`
```

Comment:
neni to duplicita k https://github.com/Drupalcz/drupalcz/blob/develop/docroot/sites/default/settings/default.local.settings.php#L83 ?